### PR TITLE
Added 4 new questions into Project Description page

### PR DESCRIPTION
**Pull request by TylerGoulskiERG**
Added 4 new questions into Project Description page.  
projectStudyArea
projectTimePeriod
projectParameters
projectDataUsers

Updates projectDescription to projectObjective

### CODE REVIEW

File: client/src/views/Navigate.vue [hunk 1]
```diff
@@ -129,7 +130,8 @@
                     <p :key="index" class="has-text-black example-text" ref="exampleText" v-html="example.text"></p>
                   </template>
                 </Tabs>
-                <p v-else class="has-text-black example-text" ref="exampleText" v-html="question.examples[0].text"></p>
+                <p v-else class="has-text-black example-text" ref="exampleText" v-html="getExampleText(question)"></p>
+                <!-- <p v-else class="has-text-black example-text" ref="exampleText" v-html="question"></p> -->
```
Comment (dpgalloway) [inline client/src/views/Navigate.vue]: Please remove this commented p tag

File: client/src/views/Navigate.vue [hunk 2]
```diff
@@ -254,6 +256,8 @@ export default {
     },
   },
   async mounted() {
+    console.log(this.currentQuestions);
```
Comment (dpgalloway) [inline client/src/views/Navigate.vue]: Remove this console.log

File: client/src/views/Navigate.vue [hunk 3]
```diff
@@ -274,6 +278,13 @@ export default {
     }
   },
   methods: {
+    getExampleText(question) {
+      console.log(question);
```
Comment (dpgalloway) [inline client/src/views/Navigate.vue]: Remove this console.log

File: client/src/views/Navigate.vue [hunk 4]
```diff
@@ -110,13 +111,13 @@
...
-                  @click.native="() => (shouldShowExample = true)"
+                  v-if="question.examples.length > 0"
+                  @click.native="() => (shouldShowExample = question)"
                 />
               </div>
-              <Modal v-if="shouldShowExample" @close="() => (shouldShowExample = false)">
+              <Modal v-if="shouldShowExample === question" @close="() => (shouldShowExample = null)">
                 <Tabs
-                  v-if="question.examples.length > 1"
+                  v-if="question.examples.length"
```
Comment (dpgalloway) [inline client/src/views/Navigate.vue]: I'm not sure if this change was intended? Would the Tabs now show up even if there's only one example? If there's only one example we wouldn't want the tabs to display

**Review by dpgalloway — CHANGES_REQUESTED** [review]

File: client/src/views/Navigate.vue (hunk 1 above)
Comment (TylerGoulskiERG) [reply client/src/views/Navigate.vue]: Removed with commit 19d813f

File: client/src/views/Navigate.vue (hunk 2 above)
Comment (TylerGoulskiERG) [reply client/src/views/Navigate.vue]: Removed with commit 19d813f

File: client/src/views/Navigate.vue (hunk 3 above)
Comment (TylerGoulskiERG) [reply client/src/views/Navigate.vue]: Removed with commit 19d813f

File: client/src/views/Navigate.vue (hunk 4 above)
Comment (TylerGoulskiERG) [reply client/src/views/Navigate.vue]: Updated with commit 19d813f

**Review by dpgalloway — APPROVED** [review]

**Pull request merged**